src/notebook.c

```diff
@@ -548,33 +548,6 @@ static gboolean notebook_tab_bar_click_cb(GtkWidget *widget, GdkEventButton *eve
 }
 
 
-static gboolean notebook_tab_bar_scroll_cb(GtkWidget *widget, GdkEventScroll *event)
-{
-	GtkNotebook *notebook = GTK_NOTEBOOK(widget);
-	GtkWidget *child;
-
-	child = gtk_notebook_get_nth_page(notebook, gtk_notebook_get_current_page(notebook));
-	if (child == NULL)
```

Comment:
Any idea what this check is supposed to do? (Just curious since the new code doesn't contain this check)